example/build.gradle

```diff
@@ -7,11 +7,12 @@ repositories {
 def android_support_version = '25.3.1'
 
 dependencies {
-    compile project(':stripe')
     compile project(':android-pay')
```

Comment:
This line will get replaced with `compile 'com.stripe:android-pay:4.0.2'` after we ship